conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /import-voluntary-disclosure-frontend              app.Routes
-->         /                          health.Routes
+->         /import-voluntary-disclosure-frontend        app.Routes
```

Comment:
Based on the. config in `mdtp-frontend-routes` this should be `/import-voluntary-disclosure`